docs/index.rst

```diff
@@ -79,47 +93,46 @@ Contents
    Module Reference <api/modules>
 
 
-Indices and tables
-------------------
-
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
-
+Notes
+-----
 
-.. [#index1] PyScaffold's project template makes use of a dedicated ``src``
```

Comment:
Having a lot of footnote marks on the text is a bit off-putting... it is like a series of buts that the user have to pay attention to not be tricky. Therefore it is more appropriate to create a single notes section at the end, with secondary/complementary information